app/app.go

```diff
@@ -1602,7 +1589,7 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 // SimulationManager implements the SimulationApp interface
 func (app *StrideApp) SimulationManager() *module.SimulationManager {
-	return app.sm
+	return nil
```

Comment:
why?

Reply:
we didn't set the sim manager (it was commented out) so i just removed it from the App struct